test/command_line/test_symmetry.py

```diff
@@ -21,3 +25,44 @@ def test_symmetry(dials_regression, tmpdir):
     assert not result.returncode and not result.stderr
     assert tmpdir.join("symmetrized.refl").check()
     assert tmpdir.join("symmetrized.expt").check()
+
+
+from dials.algorithms.symmetry.cosym._generate_test_data import (
+    generate_experiments_reflections,
+)
+
+
+def test_symmetry_R3m_I2m(tmpdir):
+    os.chdir(tmpdir.strpath)
+    space_group = "C 2"
+    unit_cell = (53.173, 61.245, 69.292, 90.0, 93.04675, 90.0)
+    space_group = sgtbx.space_group_info(space_group).group()
```

Comment:
Would `sgtbx.space_group(space_group)` not suffice here?  Equivalent but a little clearer.  I've never been entirely clear what `sgtbx.space_group_info` is really for...  It's some ancient bit of Ralf's stuff and it has no docstring.

Reply:
Oh no, hang on, there's some weirdness here:
```Python
>>> import sgtbx
>>> print(sgtbx.space_group('C 2').type().universal_hermann_mauguin_symbol())
P 1 2 1 (c,2*a+c,b)
>>> print(sgtbx.space_group_info('C 2').group().type().universal_hermann_mauguin_symbol())
C 1 2 1
```

Reply:
to the uninitiated this smells like a bug?

Reply:
Anyhow - for the purposes of the calculation the long winded route through `space_group_info` _works_ so we are not in a bad place?